src/Admin/AbstractAdmin.php

```diff
@@ -2286,6 +2214,89 @@ final protected function appendParentObject(object $object): void
         }
     }
 
+    /**
+     * @return array<string, array<string, mixed>>
+     *
+     * @phpstan-param T|null $object
+     */
+    private function getInternalActionButtons(string $action, ?object $object = null): array
```

Comment:
```suggestion
    private function getDefaultActionButtons(string $action, ?object $object = null): array
```

Reply:
@core23 Are you sure? Since the constant is also called `INTERNAL_ACTIONS` i think `getInternalActionButtons` makes more sense.

Reply:
It makes sens, if we change the implementation we won't have to change the name.

Reply:
Ok. I've changed it. I didn't use the proposed suggestion because the method call wasn't changed.